src/model/cytoscape-syncher.js

```diff
@@ -376,21 +392,23 @@ export class CytoscapeSyncher {
                   const elePos = ele.position();
 
                   if( doc.position.x !== elePos.x || doc.position.y !== elePos.y ){
-                    const syncPosAni = ele.animation({
-                      position: _.clone(doc.position),
-                      duration: 250,
-                      easing: 'ease-out'
-                    });
+                    ele.position(_.clone(doc.position));
```

Comment:
I disabled the position animations for now so that we can see the raw updates coming in re. #111